definitions/digital_analytics_domain/product_v1/aggregated_vbb.sqlx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+config {
+    type: "view",
+    schema: functions.productSchema("ga4"),
+    description: "Source data for Aggregated VBB pipeline.",
+    tags: ['ga4', 'vbb']
+}
+js {
+    const eventDateField = "event_date"
+    const selectFields = sql.aggregatedVBBColumns(eventDateField, "Dt", ga4.aggregatedVBBColumns);
+}
+SELECT
+  DISTINCT ${selectFields}
+FROM
+  ${ref({schema: functions.productSchema("ga4"), name: "event"})}
+WHERE
+  ${eventDateField} >= DATE_SUB(CURRENT_DATE(), INTERVAL 1 YEAR)
+ORDER BY
```

Comment:
We should never add ORDER BY clauses for views, or anything else for that matter - performance killers.

Reply:
Do we have a performance problem? This will ORDER 365 rows, one row per each day.

Reply:
Just trying to separate the data from the presentation/consumption. The consumer can always have "select * from some_view order by x" if they need to. I didn't realize that this view doesn't have customer or customer segment in it when I wrote the comment.